### ISSUE TITLE
CRI: Sandbox seccomp and apparmor

### ISSUE THREAD

**Issue by Random-Liu** [kubernetes/kubernetes#51288]
Currently, there is seccomp profile field in sandbox security context, but no apparmor field in it.

* If this is intended, what is the sandbox seccomp profile used for? Should we pass the seccomp profile to sandbox container in dockershim https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/security_context.go#L39;
* If this is not, we should fix it.

@yujuhong @feiskyer @mikebrow

**Comment by mikebrow** [comment]
Good questions. Could use some guidance for the expectations placed on CRI implementors for pods and containers for seccomp and apparmor.

**Comment by feiskyer** [comment]
> If this is intended

Yep, this is intended. Kubernetes API exposes pod seccomp profiles, but there is only container apparmor. (see [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/annotation_key_constants.go#L42) and [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/security/apparmor/helpers.go#L57)).
